### PR TITLE
Company page

### PR DESCRIPTION
**Pull request by douglastmp**
Addition of company's page to edit information about company in the admin.

### CODE REVIEW

**Comment by douglastmp** [comment]
It closes #37

File: pages/admin/company.vue [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    }
+  },
+  methods: {
+    check() {
+      const constraints = {
+        nameError: {
+          presence: { message: '^Nome da empresa não pode ser vazio' },
+        },
+        emailError: {
+          presence: { message: '^E-mail não pode ser vazio' },
```
Comment (allanhal) [inline pages/admin/company.vue]: Set email validation.

File: pages/admin/company.vue [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        },
+        cepError: {
+          presence: { message: '^CEP não pode ser vazio' },
```
Comment (allanhal) [inline pages/admin/company.vue]: Number/mask validation

File: pages/admin/company.vue [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+      <fieldset>
+        <label for="address">Endereço</label>
+        <!-- Company CEP -->
+        <label for="cep">CEP</label>
+        <input v-model="cep" name="cep" type="text" />
+        <p class="error">{{ errors.cepError | first }}</p>
+        <!-- Company state -->
+        <label for="state">Estado</label>
+        <select>
+          <option value=""></option>
```
Comment (carlosxjm) [inline pages/admin/company.vue]: Please, use `v-for` importing the state data from other file to make this file cleaner.

File: pages/admin/company.vue [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+      city: '',
+      neighborhood: '',
+      street: '',
+      number: '',
+    }
+  },
+  methods: {
+    check() {
+      const constraints = {
+        nameError: {
```
Comment (carlosxjm) [inline pages/admin/company.vue]: We're to the constraints keys the field name without the suffix error.

File: pages/admin/company.vue [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+    },
+    validateForm() {},
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+input[type='number']::-webkit-inner-spin-button,
+input[type='number']::-webkit-outer-spin-button {
+  -webkit-appearance: none;
```
Comment (carlosxjm) [inline pages/admin/company.vue]: CSS prefixes are added automatically, no necessary add it here.

File: pages/admin/company.vue [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+    submit() {
+      // TODO: Send to the api
+    },
+    validateForm() {},
+  },
+}
+</script>
+
+<style lang="scss" scoped>
```
Comment (carlosxjm) [inline pages/admin/company.vue]: Use sass

File: pages/admin/company.vue [hunk 7]
```diff
@@ -0,0 +1,178 @@
+<template>
+  <div class="content">
+    <h2>Editar informações</h2>
+    <form @submit.prevent="check" novalidate>
+      <!-- Company name -->
```
Comment (carlosxjm) [inline pages/admin/company.vue]: Please, remove the form comments.

**Comment by carlosxjm** [comment]
@allanhal is continuing this PR here #76

**Pull request closed without merging**